### PR TITLE
added RandomNumber and LevelUp entries.

### PR DESCRIPTION
**Pull request by vk2gpz**
### Type

- [ ] Internal change (Doesn't affect end-user).
- [ ] External change (Does affect end-user).
- [x] Wiki (Changes towards the [Wiki]).
- [ ] Other: __________ 

### Description

Closes N/A 

[Wiki]: https://github.com/PlaceholderAPI/PlaceholderAPI/wiki

### CODE REVIEW

File: wiki/Placeholders.md [hunk 1]
```diff
@@ -3001,6 +3017,35 @@ Please see [this discussion][list] for a list of all expansions officially maint
   ```
 ----
 
+- ### **[LevelUp](https://polymart.org/resource/levelup.457/)**
+  > /papi ecloud download LevelUp
+  ```
```
Comment (Andre601) [inline wiki/Placeholders.md:3022]:
````suggestion
- ### **[LevelUp](https://polymart.org/resource/457/)**
  > /papi ecloud download LevelUp
  
  ```
````

File: wiki/Placeholders.md [hunk 2]
```diff
@@ -3001,6 +3017,35 @@ Please see [this discussion][list] for a list of all expansions officially maint
...
+  %levelup_sword_xp_to_next%
+  %levelup_bow_level%
+  %levelup_bow_level_percentage%
+  %levelup_bow_level_meter%
+  %levelup_bow_xp%
+  %levelup_bow_xp_percentage%
+  %levelup_bow_xp_meter%
+  %levelup_bow_xp_to_next%
+  ```
+----
```
Comment (Andre601) [inline wiki/Placeholders.md:3047]:
```suggestion

----
```

**Review by Andre601 — CHANGES_REQUESTED** [review]

**Comment by vk2gpz** [comment]
is this some kind of joke??

**Comment by Andre601** [comment]
I'm sorry that we have guidelines stating how certain things should be.

**Pull request merged**